server/index.js

```diff
@@ -33,6 +32,21 @@ app.get('/api/products', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products/:productId', (req, res, next) => {
+  const sqlQuery = `select *
+                    from "products"
+                    where "productId" = '${req.params.productId}'`;
+
+  db.query(sqlQuery)
+    .then(result => {
+      if (!result.rowCount) {
+        next(new ClientError(`There are no records matching id: ${req.params.productId}`, 404));
+      }
+      res.json(result.rows[0]);
```

Comment:
Be sure to `return` or use an `else` to prevent the server from attempting to respond if there's no product.